src/bson_array.c

```diff
@@ -207,6 +208,144 @@ BsonArray bson_array_from_bytes(uint8_t *data) {
   return array;
 }
 
+size_t bson_array_from_bytes_len(BsonArray *output, const uint8_t *data, size_t data_size) {
+  const uint8_t *current = data;
+  size_t remain_bytes = data_size;
```

Comment:
Currently all variables are `camelCase` in this project.

Reply:
Thanks, updated with commit 17f0fd77964cc7848c12b600cec8f7cac810cbf7.